nearlib/test/integration.test.js

```diff
@@ -167,6 +167,81 @@ test('create account with a new key and then view account returns the created ac
     expect(aliceAccountAfterCreation.amount).toBe(aliceAccountBeforeCreation.amount - amount);
 });
 
+describe('with access key', function () {
+    let oldLog;
+    let logs;
+    let contractId = 'test_contract_' + Date.now();
+    let newAccountId;
+    let newAccountKeyPair;
+
+    beforeAll(async () => {
+        const keyWithRandomSeed = await KeyPair.fromRandomSeed();
+        const createAccountResponse = await account.createAccount(
+            contractId,
+            keyWithRandomSeed.getPublicKey(),
+            10,
+            aliceAccountName);
+        await nearjs.waitForTransactionResult(createAccountResponse);
+        await keyStore.setKey(contractId, keyWithRandomSeed);
+        const data = [...fs.readFileSync('../tests/hello.wasm')];
+        await nearjs.waitForTransactionResult(
+            await nearjs.deployContract(contractId, data));
+
+    });
+
+    beforeEach(async () => {
+        oldLog = console.log;
+        logs = [];
+        console.log = function() {
+            logs.push(Array.from(arguments).join(' '));
+        };
+
+        newAccountId = await generateUniqueString('create.account.test');
+        newAccountKeyPair = await KeyPair.fromRandomSeed();
+        const createAccountResponse = await account.createAccount(
+            newAccountId,
+            newAccountKeyPair.getPublicKey(),
+            0,
+            aliceAccountName);
+        await nearjs.waitForTransactionResult(createAccountResponse);
+        await keyStore.setKey(newAccountId, newAccountKeyPair);
+    });
+
+    afterEach(async () => {
+        console.log = oldLog;
+    });
+
+    test('make function calls using access key', async () => {
+        // Adding access key
+        const keyForAccessKey = await KeyPair.fromRandomSeed();
+        const addAccessKeyResponse = await account.addAccessKey(
+            newAccountId,
+            keyForAccessKey.getPublicKey(),
+            contractId,
+            '',
```

Comment:
no idea what are `''`, `''`, `0` here without going to check `addAccessKey` method.

I think it makes sense to either pass parameters as part of options object, or save them to named variables before passing (unless immediately obvious what it is)

Reply:
My IDE shows which argument it is for unnamed args. I'll add comments

Reply:
@evgenykuzyakov IDE does this, but e.g. right here we don't have this feature available. It's not like most time _reading_ code is spent inside of IDE.